.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    # Ruff version.
+    rev: v0.0.272
+    hooks:
+      - id: ruff
+        args: [ --fix, --exit-non-zero-on-fix ]
```

Comment:
remove args for now while we figure out what we want to do in the pipeline.  having both --fix and --exit-non-zero-on fix can give some backwards results.  --fix will fix it and then --exi-non-zero-on-fix will give an error even if auto correction fixes it.